packages/mdc-textfield/_mixins.scss

```diff
@@ -267,10 +284,7 @@
   }
 
   .mdc-floating-label {
-    @include mdc-rtl-reflexive-position(left, 16px);
-
-    position: absolute;
-    bottom: 20px;
```

Comment:
I'm a bit confused why this was moved to a common mixin used for both filled and outlined text fields, if it was formerly specific to outlined?

Reply:
It was specified in the outlined mixin but actually it was used in both the box variant and the outlined.